pynetgear/__init__.py

```diff
@@ -149,11 +149,15 @@ def get_attached_devices(self):
         decoded = node.text.strip().replace(UNKNOWN_DEVICE_ENCODED,
                                             UNKNOWN_DEVICE_DECODED)
 
-        if not decoded or decoded == "0":
+        if not decoded:
             _LOGGER.error("Can't parse attached devices string")
```

Comment:
Any reason to just not use info or warn here instead of error ? Because empty string and '0' string looks like the same case to me. The message is bad however, something like "Empty or 0 devices string" ?
You can also remove `_LOGGER.debug(node.text.strip())`, it's pretty useless since we know the string is empty or 0...

Reply:
I am happy you did a PR however don't take it wrong :) For small stuffs like that I try to push people to do it themselves so they see that after all, it's not that difficult to fix something and do a PR, especially with Python.

Reply:
hehe good to know :) It was easier than I thought. The reason for splitting is that the error kept coming. But I will do some more check to verify before I update.

Reply:
Even with fatal in default logger the warning pops up. So I suggest to keep the split so we still have the error if it is not decoded, which maybe indicates something worse?
```suggestion
        if not decoded:
            _LOGGER.error("Can't parse attached devices string")
            return devices
            
        if decoded == "0":
            _LOGGER.info("Empty or 0 devices string")
            return devices
```

Reply:
Even with fatal in the default logger (HA) it pops up. Based on this I suggest to split and keep the error on the "if not decoded" part. This way we keep errors on something indicating something worse?

Reply:
Not really needed, the not in this case just test for an empty string so it is not really an error case either. Just squash that in one info please it's more than enough ;)

Reply:
Done. A simple change then. Hopefully its OK now :)